### PR TITLE
Add output_format arg, that provides for JSON output of build status

### PR DESCRIPTION
**Pull request by timhaines**
Thanks @joscha for doing the majority of the work here in https://github.com/percy/percy-storybook/pull/19.

Also includes a fix for https://github.com/percy/percy-storybook/issues/20.

### CODE REVIEW

**Comment by joscha** [comment]
What do you think about making the format parameter lowercase or accepting it case-independent.

**Comment by timhaines** [comment]
@joscha Done.  output_format now accepts JSON or json.

File: packages/percy-storybook/src/cli.js [hunk 1]
```diff
@@ -66,13 +74,19 @@ export async function run(argv) {
   // debug('assets %o', assets);
 
   const stories = await getStories(assets[storybookJavascriptPath], options);
-  debug('stories %o', stories);
+  options.debug && debug('stories %o', stories);
```
Comment (fotinakis) [inline packages/percy-storybook/src/cli.js]: This toggling of output should be the job of the `debug` function itself, I believe you can just do `debug.enabled = options.debug` to toggle it on or off. https://www.npmjs.com/package/debug#checking-whether-a-debug-target-is-enabled

File: packages/percy-storybook/src/getOutputFormat.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default function getOutputFormat(outputFormatString) {
```
Comment (fotinakis) [inline packages/percy-storybook/src/getOutputFormat.js]: Maybe `normalizeOutputFormat`?

**Pull request merged**